tests/test_container.py

```diff
@@ -87,7 +87,6 @@ class Something(object):
 
     Something([1, 2, 3, 4])
 
-
```

Comment:
Can you please add back this removed line ?

Reply:
sure, done